Source/WebCore/PAL/pal/graphics/WebGPU/WebGPUExternalTextureDescriptor.h

```diff
@@ -35,22 +36,14 @@ typedef struct __CVBuffer* CVPixelBufferRef;
 
 namespace PAL::WebGPU {
 
-struct HTMLVideoElementIdentifier {
-    uint64_t identifier;
-};
-struct WebCodecsVideoFrameIdentifier {
-    std::pair<uint64_t, uint64_t> identifier;
-};
-
-using VideoSourceIdentifier = std::variant<HTMLVideoElementIdentifier, WebCodecsVideoFrameIdentifier>;
-
 struct ExternalTextureDescriptor : public ObjectDescriptorBase {
-    VideoSourceIdentifier mediaIdentifier;
+    uint64_t mediaIdentifier;
```

Comment:
I think here it should really help crystallizing the problem for your thought when you would define this, too as
```
using ExternalTextureSource = std::variant</* what is this??? */>;

struct ExternalTextureDescriptor : public ObjectDescriptorBase {
    PredefinedColorSpace colorSpace { PredefinedColorSpace::SRGB };
    ExternalTextureSource source;
};
```